src/Marten.Testing/Examples/ForeignKeyExamples.cs

```diff
@@ -20,6 +20,32 @@ public void configuration()
 
             //var sql = store.Schema.ToDDL();
             //Console.WriteLine(sql);
+        }
+
+        public void external_fkey()
+        {
+            // SAMPLE: configure-external-foreign-key
+var store = DocumentStore.For(_ =>
```

Comment:
You don't have to left justify code samples anymore. I fixed that in stdocs quite awhile back.